__tests__/app.test.js

```diff
@@ -33,3 +33,30 @@ describe('GET /api/topics', () => {
     });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+  test('Status 200: Responds with an article from the DB based on the id parameter in the request', () => {
+    return request(app).get('/api/articles/8').expect(200).then(({body})=>{
+      const article = {
+        article_id: 8,
+        title: 'Does Mitch predate civilisation?',
+        topic: 'mitch',
+        author: 'icellusedkars',
+        body: 'Archaeologists have uncovered a gigantic statue from the dawn of humanity, and it has an uncanny resemblance to Mitch. Surely I am not the only person who can see this?!',
+        created_at: expect.any(String),
+        votes: 0,
+      };
+      expect(body.article).toEqual(article);
+    });
+  });
+  test('Status 400: responds with "Invalid Input" if article_id is not in the correct format', () => {
+    return request(app).get('/api/articles/handbag').expect(400).then(({body})=>{
+      expect(body.msg).toEqual('Invalid input');
```

Comment:
when matching primitives try to get the hang of using `toBe` rather than `toEqual`